ETC.tsv

```diff
@@ -10044,7 +10044,7 @@ ETC_20150323_010045	{memo X}had the item repaired.
 ETC_20150323_010046	$Collecting Young Leaves
 ETC_20150323_010047	$Collected a Young Leaf, but the tough leaf cut you!
 ETC_20150323_010048	$The tough leaves cut your hand!
-ETC_20150323_010049	You threw the music box back to where it was
+ETC_20150323_010049	You threw the music box to where it was
```

Comment:
"to where it was" alone lacks something.  You don't say that in natural English.  You'd say "to where it was obtained" or "back to where it was"
If you want it shorter, you could say "You threw the music box back"